dvc/utils/__init__.py

```diff
@@ -212,3 +212,10 @@ def convert_to_unicode(data):
         return type(data)(map(convert_to_unicode, data))
     else:
         return data
+
+
+def tmp_postfix():
+    """ Temporary name for a partial download """
+    from uuid import uuid4
+
+    return "-" + str(uuid4()) + ".tmp"
```

Comment:
Why do we need double suffix though?

Reply:
I like how you can glob your `*.tmp` files, while keeping the "randomness" of having the `uuid`.

How about `filename.uuid4.tmp`? Do you prefer the previous one?  :)

Reply:
@mroutis That is reasonable, I do like `filename.uuid4.tmp` :slightly_smiling_face: Thank you!